articles/libraries/machine-learning/design.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Quantum machine learning library
+description: Learn the basic concepts of designing circuit models for the quantum circuit centric classifier.
+author: geduardo
+ms.author: v-edsanc@microsoft.com
+ms.date: 02/17/2020
+ms.topic: article
+uid: microsoft.quantum.libraries.machine-learning.design
+---
+
+# Design your own classifier
+
+In this tutorial you will learn the basic concepts behind the design of circuit models for the quantum circuit centric classifier.
```

Comment:
Or similar. A tutorial is meant to be hands on

Reply:
Is this really a how to guide?

Reply:
What is the intent of this article?

Reply:
Rewording to "guide"